lua/lspconfig/server_configurations/svelte.lua

```diff
@@ -1,3 +1,4 @@
+local root_files = {'package.json', '.git'}
```

Comment:
```suggestion
local root_files = { 'package.json', '.git' }
```

Reply:
I skipped running the formatter because it was modifying other parts, I can fix the macro I'm using instead tho.